### DISCUSSION TITLE
Why is P Sclice missing in my output?

### DISCUSSION THREAD

**Discussion by Pezeo** [fraunhoferhhi/vvenc#231] [Q&A]
Here's the output detail.

SUMMARY --------------------------------------------------------
        Total Frames |   Bitrate     Y-PSNR    U-PSNR    V-PSNR    YUV-PSNR
             1174    a    2552.1377   10.7738   10.5712   10.5767   10.6292

I Slices--------------------------------------------------------
        Total Frames |   Bitrate     Y-PSNR    U-PSNR    V-PSNR    YUV-PSNR
               37    i   80699.3050   17.2293   13.5518   13.5473   15.5626

P Slices--------------------------------------------------------
        Total Frames |   Bitrate     Y-PSNR    U-PSNR    V-PSNR    YUV-PSNR
                0    p    -nan(ind)  -nan(ind)  -nan(ind)  -nan(ind)  -nan(ind)

B Slices--------------------------------------------------------
        Total Frames |   Bitrate     Y-PSNR    U-PSNR    V-PSNR    YUV-PSNR
             1137    b       9.0901   10.5637   10.4743   10.4800   10.5343

**Comment by adamjw24** [comment]
See https://github.com/fraunhoferhhi/vvenc/discussions/137#discussioncomment-3173856
